src/arrays.js

```diff
@@ -1,61 +1,61 @@
 const getNthElement = (index, array) => {
-  // your code here
-};
-
-const arrayToCSVString = (array) => {
-  // your code here
+  if (array.length <= 4) {
+    return array[index];
+  }
+  };
+const arrayToCSVString = (array) => { 
+  return array.join()
 };
 
 const csvStringToArray = (string) => {
-  // your code here
+  return string.split(',');
 };
 
 const addToArray = (element, array) => {
-  // your code here
+  array.push(element);
 };
 
 const addToArray2 = (element, array) => {
-  // your code here
+  array.concat(element);
```

Comment:
this line is probably unnecessary. If you're not assigning the new array to a variable then JS forgets it pretty much instantly.